sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,10 +1,23 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
+
+### New Features
+
+* Support for replaceable HTTP transport layer.
```

Comment:
Is this the new `TransportPolicyOptions` member as part of client options?

Reply:
yep